models/notification_test.go

```diff
@@ -16,8 +16,12 @@ func TestCreateOrUpdateIssueNotifications(t *testing.T) {
 
 	assert.NoError(t, CreateOrUpdateIssueNotifications(issue, 2))
 
-	// Two watchers are inactive, thus only notification for user 10 is created
-	notf := AssertExistsAndLoadBean(t, &Notification{UserID: 10, IssueID: issue.ID}).(*Notification)
+	// 9 are inactive, thus notifications for user 1 and 4 is created
+	notf := AssertExistsAndLoadBean(t, &Notification{UserID: 1, IssueID: issue.ID}).(*Notification)
+	assert.Equal(t, NotificationStatusUnread, notf.Status)
+	CheckConsistencyFor(t, &Issue{ID: issue.ID})
```

Comment:
nit: only need to call this once, preferably at the end of the test (for consistency with other tests)

Reply:
done.